pyproject.toml

```diff
@@ -0,0 +1,82 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.mypy]
+warn_return_any = true
+warn_unused_configs = true
+
+[tool.poetry]
+authors = ["Elixir Cloud AAI <cloud-service@elixir-europe.org>"]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "License :: OSI Approved :: Apache Software License",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.11",
+]
+description = "File Handler Utilizing TUS and Minio (S3 Storage) with DRS Filer Integration"
+license = "Apache-2.0"
+maintainers = ["Elixir Cloud AAI <cloud-service@elixir-europe.org>"]
+name = "tus_storagehandler"
+readme = "README.md"
+repository = "https://github.com/elixir-cloud-aai/tus-storagehandler"
+version = "0.1.0"
+
+[tool.poetry.dependencies]
+python = "^3.11"
```

Comment:
change it to `python = ">= 3.11<4.0.0"`, right @uniqueg?\

Because python is largely backwards compatible but if we leave it to python = "^3.11", and someone with global installation of python 3.12 tries to run the proj, poetry won't allow it.

Reply:
In fact, there likely won't be a Python 4 - or at least not within the next couple of years. At least that's what I'm reading.